### PR TITLE
Add a bio for @githubteacher

### PR DESCRIPTION
**Pull request by githubteacher**
This Pull Request adds a bio for @githubteacher as requested in #6.  

Hey @githubstudent, I would :heart: if you had a few :watch: to put some :eyes: on this.

### CODE REVIEW

**Comment by githubteacher** [comment]
These changes look :ok:, but I will add a few line comments.

File: docs/githubteacher.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# GitHub Teacher Bio
+
+**Name:** GitHub Teacher
+
+**Occupation:** Teacher
+
+**Location:** San Francisco
```
Comment (githubteacher) [inline docs/githubteacher.md]: I think we might want to be more specific here.  There are lots of San Franciscos in the :earth_americas:

**Comment by sikka** [comment]
Changes look great!

**Comment by manirs76** [comment]
I accidentally did the merge as well, so mine is in "closed" state

**Comment by githubteacher** [comment]
:+1: 
:ship: it!

:shipit:

**Pull request merged**